File: lab3/class.py
import random


class studentIDList:
    def __init__(self):
        self.numbers = []

        for i in range(1000):
            random_number = random.randint(1, 1000)
            self.numbers.append(random_number)

    def show_numbers(self):
        for number in self.numbers:
            print(number)


def main():
    student_list = studentIDList()
    student_list.show_numbers()


if __name__ == "__main__":
    main()

File: lab3/class.cpp
#include <iostream>
#include <cstdlib>

using namespace std;

class studentIDList {
public:
    studentIDList();
    void show_numbers();

private:
    int numbers[1000];
};


studentIDList::studentIDList() {
    for (int i = 0; i < 1000; i++) {
        numbers[i] = rand() % 1000 + 1;
    }
}


void studentIDList::show_numbers() {
    for (int i = 0; i < 1000; i++) {
        cout << numbers[i] << endl;
    }
}


int main() {
    studentIDList student_list1;
    student_list1.show_numbers();

    return 0;
}